composer.json

```diff
@@ -53,6 +53,17 @@
             "web/profiles/contrib/{$name}": ["type:drupal-profile"],
             "web/themes/contrib/{$name}": ["type:drupal-theme"],
             "drush/contrib/{$name}": ["type:drupal-drush"]
+        },
+        "patches": {
+            "drupal/ctools": {
+                "Add permission condition": "https://www.drupal.org/files/issues/2665692-ctools-6.patch"
```

Comment:
This question is not necessarily needed in this case since the issue-number is in the file name of the patch. But should we prefix the description with the issue? Ex:

`"2665692: Add permission condition": "https://www.drupal.org/files/issues/2665692-ctools-6.patch"`

This is, currently, not a problem. But if we use a patch that doesn't follow the _SUGGESTED_ naming convention Drupal provides when uploading a patch in an issue queue, then we're kind of on our own if we want to reverse-engineer where it comes from so we can check for updated/status when updating the module to a new version.

What do you think @kasperg ?